beacon-chain/blockchain/process_block_test.go

```diff
@@ -3244,3 +3244,349 @@ func TestSaveLightClientBootstrap(t *testing.T) {
 
 	reset()
 }
+
+func setupLightClientTestRequirements(ctx context.Context, t *testing.T, s *Service, v int, options ...util.LightClientOption) (*util.TestLightClient, *postBlockProcessConfig) {
+	var l *util.TestLightClient
+	switch v {
+	case version.Altair:
+		l = util.NewTestLightClient(t, version.Altair, options...)
+	case version.Bellatrix:
+		l = util.NewTestLightClient(t, version.Bellatrix, options...)
+	case version.Capella:
+		l = util.NewTestLightClient(t, version.Capella, options...)
+	case version.Deneb:
+		l = util.NewTestLightClient(t, version.Deneb, options...)
+	case version.Electra:
+		l = util.NewTestLightClient(t, version.Electra, options...)
+	default:
+		t.Error("Invalid fork version")
```

Comment:
```suggestion
		t.Errorf("Unsupported fork version %s", version.String(v))
```
`Invalid` feels like the fork doesn't exist but it may be a valid fork (e.g. Fulu)